pages/instructorAssessmentManualGrading/instructorAssessmentManualGrading.ejs

```diff
@@ -26,7 +26,12 @@
                 <th></th>
                 <th>QID</th>
                 <th>Points</th>
-                <th>Number of Submissions</th>
+                <th>Total Submissions</th>
+                <th>Last Submissions for Manual Grading</th>
+                <th>Manually Graded</th>
+                <th>Locked Submissions</th>
```

Comment:
Do we care about this?

Reply:
I don't think we do... The idea of a "locked submission" is a little risky imo. I think the sync issues should be handled via communication/warnings, rather than locks, anyway.